packages/vuetify/src/components/VDatePicker/__tests__/VDatePicker.date.spec.ts

```diff
@@ -633,4 +633,26 @@ describe('VDatePicker.ts', () => { // eslint-disable-line max-statements
     wrapper.findAll('.v-date-picker-table--date tbody tr+tr td:first-child button').at(0).trigger('dblclick')
     expect(dblclick).toHaveBeenCalledWith('2013-05-05')
   })
+
+  it('should handle date range select', async () => {
+    const cb = jest.fn()
+    const wrapper = mountFunction({
+      propsData: {
+        multiple: true,
+        selectRange: true,
+        value: ['2019-01-01', '2019-01-02'],
+      },
+    })
+
+    wrapper.vm.$on('input', cb)
+    wrapper.findAll('.v-date-picker-table--date tbody tr+tr td:first-child button').at(0).trigger('click')
+    expect(cb.mock.calls[0][0]).toEqual(
+      expect.arrayContaining(['2019-01-06'])
+    )
+
+    wrapper.findAll('.v-date-picker-table--date tbody tr+tr td button').at(2).trigger('click')
+    console.log(JSON.stringify(cb.mock.calls))
+    expect(cb.mock.calls[0][0][0]).toBe('2019-01-06')
+    expect(cb.mock.calls[1][0][0]).toBe('2019-01-08')
```

Comment:
why? first date should equal `2019-01-01`, shouldn't it?

Reply:
I am trying to use the array of length 2 to keep the [From ,To] range. So further clicking will wipe the previous range.
['2019-01-01', '2019-01-02'] => first click => ['2019-01-06'] => second click = ['2019-01-06', '2019-01-08']